WordPress/src/main/java/org/wordpress/android/ui/compose/components/menu/dropdown/MenuButton.kt

```diff
@@ -0,0 +1,121 @@
+package org.wordpress.android.ui.compose.components.menu.dropdown
+
+import android.content.res.Configuration
+import androidx.compose.animation.animateContentSize
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.R
+import org.wordpress.android.ui.compose.components.menu.dropdown.MenuElementData.Item
+import org.wordpress.android.ui.compose.theme.AppTheme
+import org.wordpress.android.ui.compose.unit.Margin
+
+@Composable
+fun DropdownMenuButton(
```

Comment:
Discussion here: https://github.com/wordpress-mobile/WordPress-Android/pull/19679#discussion_r1424608359

Reply:
https://github.com/wordpress-mobile/WordPress-Android/pull/19679#discussion_r1425965128

Reply:
Just confirming: this composable will be kept in this separate file instead of moving into `JetpackDropdownMenu.kt`? If that's the case can you rename the file to match the Composable name (`DropdownMenuButton.kt`)?